src/scripts/createUser.js

```diff
@@ -0,0 +1,67 @@
+/*
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+const readline = require('readline');
+const bcrypt = require('bcryptjs');
+const User = require('../server/models/User');
+
+const rl = readline.createInterface({
+	input: process.stdin,
+	output: process.stdout
+});
+
+function askEmail() {
+	return new Promise((resolve, reject) => {
+		rl.question('Email: ', email => {
+			// See https://stackoverflow.com/a/46181/5116950
+			// eslint-disable-next-line
+			const regexEmail = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
+			if (regexEmail.test(email))	resolve(email);
+			else reject(email);
+		});
+	});
+}
+
+function askPassword(email) {
+	return new Promise(resolve => {
+		rl.question('Password: ', password => { resolve([email, password]); });
+	});
+}
+
+function terminateReadline(message) {
+	if (message) console.log(message);
+	rl.close();
+	process.exit(0);
+}
+
+(async () => {
+	let emailResult;
+	try {
+		emailResult = await askEmail();
+	} catch (err) {
+		console.log('Invalid email, no user created');
+		rl.close();
+		process.exit(0);
+	}
+	const output = await askPassword(emailResult);
+	const email = output[0];
+	const password = output[1];
+
+	if (password.length < 8) {
+		terminateReadline('Weak password, no user created');
```

Comment:
You should tell the user beforehand that their password must be at least 8 characters long, and this error message should say that the password must be at least 8 characters long.

Reply:
Good point on the prompt. I'm not sure if we want to remove the rules entirely, it was simply a check to prevent passwords like `password`

Reply:
I think the length requirement is fine, and if you want to go any further (which I think is overkill for now) you should follow the [NIST guidelines](https://en.wikipedia.org/wiki/Password_policy#NIST_Guidelines).